### PR TITLE
Updating order book attributes optimization

### PR DESCRIPTION
**Pull request by wer1st**

### CODE REVIEW

File: pallets/order-book/src/tests/test_utils.rs [hunk 1]
```diff
@@ -328,3 +328,20 @@ pub fn run_to_block(n: u32) -> Weight {
...
+pub fn update_orderbook_unchecked(
+    order_book_id: OrderBookId<AssetIdOf<Runtime>, DEXId>,
+    tick_size: Balance,
+    step_lot_size: Balance,
+    min_lot_size: Balance,
+    max_lot_size: Balance,
+) -> OrderBook<Runtime> {
+    let mut order_book = OrderBookPallet::order_books(order_book_id).unwrap();
+    order_book.tick_size.set(tick_size);
+    order_book.step_lot_size.set(step_lot_size);
```
Comment (bragov4ik) [inline pallets/order-book/src/tests/test_utils.rs:341]: this doesn't perform any alignments, correct?

Comment (bragov4ik) [reply pallets/order-book/src/tests/test_utils.rs:341]: just want to make sure it's intended

File: pallets/order-book/src/lib.rs [hunk 2]
```diff
@@ -169,12 +168,7 @@ pub mod pallet {
         type SyntheticInfoProvider: SyntheticInfoProvider<Self::AssetId>;
         type DexInfoProvider: DexInfoProvider<Self::DEXId, DEXInfo<Self::AssetId>>;
         type Time: Time;
-        type ParameterUpdateOrigin: EnsureOrigin<
-            Self::RuntimeOrigin,
-            Success = Either<Self::AccountId, ()>,
-        >;
-        type StatusUpdateOrigin: EnsureOrigin<Self::RuntimeOrigin, Success = ()>;
-        type RemovalOrigin: EnsureOrigin<Self::RuntimeOrigin, Success = ()>;
+        type PermissionOrigin: EnsureOrigin<Self::RuntimeOrigin, Success = ()>;
```
Comment (bragov4ik) [inline pallets/order-book/src/lib.rs:171]: It seems something like `PermittedOrigin` would be more accurate. I might be misunderstanding the intention here; What do you think?

**Review by bragov4ik — APPROVED** [review]
LGTM

File: pallets/order-book/src/tests/test_utils.rs (hunk 1 above)
Comment (wer1st) [reply pallets/order-book/src/tests/test_utils.rs:341]: Yes, without alignment. Just set raw values without any checks. It's necessary to make the preconditions setup more easy for some tests.

**Review by vovac12 — APPROVED** [review]
LGTM

**Pull request merged**